doc/man3/EVP_EncryptInit.pod

```diff
@@ -251,6 +255,15 @@ EVP_CipherFinal_ex(). In previous releases they also cleaned up
 the B<ctx>, but this is no longer done and EVP_CIPHER_CTX_clean()
 must be called to free any context resources.
 
+EVP_Cipher() encrypts or decrypts a maximum I<inl> amount of bytes from
+I<in> and leaves the result in I<out>.
+If the cipher doesn't have the flag B<EVP_CIPH_FLAG_CUSTOM_CIPHER> set,
+then I<inl> must be a multiple of EVP_CIPHER_block_size().  If it isn't,
+the result is undefined.  If the cipher has that flag set, then I<inl>
+can be any size.
+This function is historic and shouldn't be used in an application, please
+consider using Update and Final functions instead.
```

Comment:
Better to reference the  full function names

Reply:
Ugh...  it's a gazillion!

Reply:
I restrained myself...  and I'll ask you to review the change.

Reply:
How about this:
```
This function is historic and shouldn't be used in an application.
Use the "update" and "final" functions (e.g., L<DESUpdate(3)>, L<DESFinal(3)>)
insead.
```

Reply:
Why the DES functions?????

Reply:
it was a sample.  pick whatever you want. :)